tasks/deploy/config/coordinator.sample.env

```diff
@@ -0,0 +1,9 @@
+NETWORK_NAME=<sepolia | base_sepolia | mainnet>
+NAME=<some name>_COORDINATOR
+PREFIX=<ERC4626 | StETH | RETH | EzETH>
+
+# needed for non-ERC4626, can be removed otherwise
+VAULT_SHARES_TOKEN_ADDRESS=0x...
```

Comment:
Let's add a comment explaining how it's used. Also, the comment style everywhere else in the codebase is to use proper capitalization with punctuation. I haven't harped on this much recently, but I'd like us to start doing this more consistently. This should be (without the change I'm asking for):

```
# Needed for non-ERC4626 coordinators. This can be removed otherwise.
```